lldb/include/lldb/Target/DynamicLoader.h

```diff
@@ -329,6 +330,11 @@ class DynamicLoader : public PluginInterface {
   /// safe to call certain APIs or SPIs.
   virtual bool IsFullyInitialized() { return true; }
 
+  /// Return the `start` function \b symbol in the dynamic loader module.
```

Comment:
The meaning of this is clear in a Darwin environment, but I wonder if we should explain it a little more, like "This is the symbol the process will begin executing with `process launch --stop-at-entry`".  I don't feel strongly about this.